### ISSUE TITLE
Add test data to development VM

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#1662]
This is one of several developer workflow issues. We should have some test data and accounts loaded upon provisioning of the development VM. This prevents a developer from doing the "make a test admin account, make a test journalist account, load the source interface and submit a bunch of documents" dance every time she wants to manually test something.

**Comment by conorsch** [comment]
Agreed! We could also automatically start the development interfaces, so the forwarded ports are accessible right out of the gate. 

Related: in the context of #749, I'm considering generating prod-specific.yml files so prod environments can be provisioned in an unattended manner, as well—doing so would greatly aid in fleshing out the CI pipeline (#1067).
